### PR TITLE
add global cors and add 2 databases

### PR DESCRIPTION
**Pull request by RomanukAlex**
- add global CORS condig;
- add application.properties for 2 databases (MySQL and PostgreSQL);
- correct rest and dao tests.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,10 +22,10 @@ buildscript {
     }
 }
 
-    buildScan {
-        termsOfServiceUrl = 'https://gradle.com/terms-of-service'
-        termsOfServiceAgree = 'yes'
-    }
+//    buildScan {
```
Comment (aleksieiev) [inline build.gradle]: We do not need it? 
remove it.

File: rest-app/src/main/java/com/epam/repair/rest/config/WebConfig.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class WebConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**")
+                .allowedOrigins("http://localhost:8090")
```
Comment (aleksieiev) [inline rest-app/src/main/java/com/epam/repair/rest/config/WebConfig.java]: looks like a great candidate to be replaced to the properties

File: rest-app/src/main/java/com/epam/repair/rest/config/WebConfig.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+                .allowedMethods("GET", "POST", "PUT", "DELETE")
+                .allowedHeaders("*")
+                .exposedHeaders("")
+                .allowCredentials(false)
+                .maxAge(3600);
```
Comment (aleksieiev) [inline rest-app/src/main/java/com/epam/repair/rest/config/WebConfig.java]: the same - property

File: service/src/main/java/com/epam/repair/service/RepairOrderServiceImpl.java [hunk 4]
```diff
@@ -51,9 +51,9 @@ public RepairOrder update(RepairOrder repairOrder) {
     }
 
     @Override
-    public void deleteById(Integer orderId) {
-        repairOrderDao.findById(orderId)
-                .orElseThrow(() -> new RuntimeException(ORDER_NOT_FOUND + orderId));
-        repairOrderDao.deleteById(orderId);
+    public void deleteById(Integer repairOrderId) {
+//        repairOrderDao.findById(repairOrderId)
```
Comment (aleksieiev) [inline service/src/main/java/com/epam/repair/service/RepairOrderServiceImpl.java]: useless comments

File: dao-api/src/main/java/com/epam/repair/dao/RepairOrderDao.java [hunk 5]
```diff
@@ -2,8 +2,14 @@
...
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface RepairOrderDao extends JpaRepository<RepairOrder, Integer> {
-}
+
+    @Modifying
+    @Query(value="delete from RepairOrder ro where ro.repairOrderId = ?1")
+    void deleteById(Integer repairOrderId);
```
Comment (aleksieiev) [inline dao-api/src/main/java/com/epam/repair/dao/RepairOrderDao.java]: What for?
@sberdachuk-epam do you know anything about this?

**Comment by RomanukAlex** [comment]
Commit fix:
- correct global CORS - replaced to the properties;
- correct deleteById (+module Model);
- add existsById in RepairOrderServiceImpl.class --> deleteById.

**Review by sergeyarhutik — APPROVED** [review]

**Pull request closed without merging**